application/android/app/src/main/java/com/group12/beabee/views/MainStructure/BaseContainerFragment.java

```diff
@@ -143,6 +152,27 @@ private void ModeEditable() {
         btnApprove.setVisibility(View.GONE);
         btnEdit.setVisibility(View.VISIBLE);
         btnAdd.setVisibility(View.GONE);
+        btnSearch.setVisibility(View.GONE);
+    }
+
+    private void ModeOnlyTitle() {
+        topbar.setVisibility(View.VISIBLE);
+        btnBack.setVisibility(View.GONE);
+        btnCancel.setVisibility(View.GONE);
+        btnApprove.setVisibility(View.GONE);
+        btnEdit.setVisibility(View.GONE);
+        btnAdd.setVisibility(View.GONE);
+        btnSearch.setVisibility(View.GONE);
+    }
+
+    private void ModeOnlySearch() {
```

Comment:
Only Search can be used both in user @ihsangkcl and market place search, thanks.